index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div>
+      <header>
+        <div class="head">
+          <p class="log">My Portfolio</p>
+          <img
+            class="hum"
+            src="photo/Icon - Menu.png"
+            alt=""
+          />
+        </div>
+      </header>
+      <div class="main">
+        <h2>
```

Comment:
- [ ] kindly use an `h1` tag here this will help google (and other search engines) knows about a specific topic